### ISSUE TITLE
Poor scrolling control in side panel on touchpad

### ISSUE THREAD

**Issue by tomsmekens** [musescore/MuseScore#24963]
### Issue type

UX/Interaction bug (incorrect behaviour)

### Description with steps to reproduce

1. In a score, make it so that any of the dialogs in the side panel (palette, properties etc.) expands beyond the dimensions of the screen and requires scrolling to navigate.
2. Use a laptop touchpad to scroll up and down.
3. Observe the user friendliness of the scrolling.

Actual behavior:
Keeping your fingers still does not stop the scrolling. It is difficult to navigate the panel because you will constantly overshoot where you actually want to be.

Expected behavior:
Stopping movement with your fingers should stop the scrolling as well. That is how it usually works in programs - firefox is one example - and it's also how it works when scrolling through the score in MuseScore itself.

### Supporting files, videos and screenshots

[Screencast from 29-09-24 03:17:03.webm](https://github.com/user-attachments/assets/e133d023-2971-40a5-a39f-6a75c3682332)

### What is the latest version of MuseScore Studio where this issue is present?

OS: Ubuntu 22.04.5 LTS, Arch.: x86_64, MuseScore Studio version (64-bit): 4.4.2-242570931, revision: github-musescore-musescore-3130f97

### Regression

No.

### Operating system

Ubuntu 22.04

### Additional context

In the attached video, I am keeping my fingers on the touchpad the whole time. I vaguely recall that this issue cropped up starting with MuseScore 3 and has persisted all this time. I'd hoped that the new Qt would fix it, but no luck.

My model is a Dell Inc. Latitude E5470 but I've had the same issue on previous laptops from Asus.

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by krasko78** [comment]
I agree. It is especially annoying for me on the palettes. Maybe we could set `maximumFlickVelocity` and `flickDeceleration` on the flickables / listviews? For me, values in the range of 800-1500 for `maximumFlickVelocity` and 10000-25000 for `flickDeceleration` seem to work best with my preference (from some quick testing) being `maximumFlickVelocity` = 1200 and `flickDeceleration` = 25000.

**Comment by bkunda** [comment]
Related to https://github.com/musescore/MuseScore/issues/24089

**Comment by cbjeukendrup** [comment]
As discussed at https://github.com/musescore/MuseScore/pull/27117, this issue doesn't apply anymore, as we've switched to Qt 6.9 (which has different scrolling problems, but those have been fixed as well).
